python/tank/pipelineconfig.py

```diff
@@ -109,8 +108,28 @@ def __init__(self, pipeline_configuration_path):
         else:
             self._bundle_cache_fallback_paths = []
 
-        # Populate the global yaml_cache if we find a pickled cache
-        # on disk.
+        # look for environment variables
+        if constants.ENV_VAR_EXTERNAL_PIPELINE_CONFIG_DATA in os.environ:
```

Comment:
Values here are overriding values in the metadata file. Maybe a quick note about why that is?